boards/risc-v/qemu-rv/rv-virt/configs/nsh64/defconfig

```diff
@@ -65,6 +66,7 @@ CONFIG_STACK_COLORATION=y
 CONFIG_START_MONTH=12
 CONFIG_START_YEAR=2021
 CONFIG_SYSTEM_NSH=y
+CONFIG_SYSTEM_NSH_STACKSIZE=3072
```

Comment:
Please change to CONFIG_INIT_STACKSIZE=3072

Reply:
@no1wudi 
I've already changed the CONFIG_INIT_STACKSIZE.

Reply:
CONFIG_ARCH_INTERRUPTSTACK seems restored to 2048

Reply:
@no1wudi 
As long as I tested, 2048 is enough for CONFIG_ARCH_INTERRUPTSTACK.